pkg/cmd/dev/generate.go

```diff
@@ -47,7 +47,7 @@ func makeGenerateCmd(runE func(cmd *cobra.Command, args []string) error) *cobra.
         dev generate execgen       # execgen targets (subset of 'dev generate go')
         dev generate schemachanger # schemachanger targets (subset of 'dev generate go')
         dev generate stringer      # stringer targets (subset of 'dev generate go')
-        dev generate testlogic     # logictest generated code (subset of 'dev generate bazel')
+        dev generate testlogic     # logictest generated code (subset of 'dev generate bazel', includes 'dev generate schemachanger')
```

Comment:
Do me a favor and remove the "subset of `dev generate bazel`" thing as it's no longer accurate.